app/views/errors/any.html.erb

```diff
@@ -1,11 +1,9 @@
-<% content_for :application_body do %>
```

Comment:
Ah you know what... then this is why the API docs also aren't formatted correctly (should be syntax highlighting of the schemas, at least).  That also uses `content_for :application_body` which does not go anywhere (see `views/layouts/application_body_api_docs.html.erb`).